packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/async-complex-params/output.js

```diff
@@ -0,0 +1,9 @@
+new class {
+  foo(_x, _x2) {
+    return babelHelpers.asyncToGenerator(function* (a, {
+      b
+    }) {})();
```

Comment:
It looks like the parameters are not being passed?

Reply:
Thank you, it has been fixed.